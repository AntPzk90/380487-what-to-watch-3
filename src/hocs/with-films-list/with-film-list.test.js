import React from 'react';
import renderer from 'react-test-renderer';
import withFilmsList from './with-films-list.jsx';

const filmMock = {
  title: `Bohemian rhapsody`,
  src: `bohemian-rhapsody.jpg`,
  poster: `bohemian-rhapsody.jpg`,
  titlePoster: `bohemian-rhapsody.jpg`,
  genre: `Comedy`,
  releaseDate: `2015`,
  preview: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
};

it(`SnapshotTest withFilmsList`, () => {

  const tree = renderer
  .create(
      <withFilmsList
        film = {filmMock}
        onCardMouseEnter = {() => {}}
      />
  )
  .toJSON();

  expect(tree).toMatchSnapshot();
});
