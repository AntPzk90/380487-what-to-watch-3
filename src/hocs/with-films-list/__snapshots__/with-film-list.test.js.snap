// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapshotTest withFilmsList 1`] = `
<withFilmsList
  film={
    Object {
      "genre": "Comedy",
      "poster": "bohemian-rhapsody.jpg",
      "preview": "https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4",
      "releaseDate": "2015",
      "src": "bohemian-rhapsody.jpg",
      "title": "Bohemian rhapsody",
      "titlePoster": "bohemian-rhapsody.jpg",
    }
  }
  onCardMouseEnter={[Function]}
/>
`;
